src/store/getters.ts

```diff
@@ -6,7 +6,8 @@ import {
 import { DateUtilities, EnumUtilities } from '@/services'
 
 export default {
-  /** The user's Keycloak GUID. */
+  /** The user's Keycloak GUID.
+   * DEPRECATED - use authentication/getKeycloakGuid() instead */
```

Comment:
How much extra work is it to change calls to `getUserKeycloakGuid` to `authentication/getKeycloakGuid` instead? (Less tech debt.)

Reply:
That's a good idea.  It's only used in a couple of places.  I'll fix it.

Reply:
Also `isRoleStaff`, please.

Reply:
Ah, good idea, I'll replace `isRoleStaff` too.

Reply:
@severinbeauvais I need to think about how to test this change.  At present, Jest loads either a fake user token or a fake staff token than calls to `isRoleStaff` are successful.  When I point the `isRoleStaff` getter to the rootState many tests break.  Not sure whether it's best to mock the getter or if there's a better way to do this.  Any suggestions?

Reply:
I think I mocked this store getters in a recent PR -- much cleaner than having a test KC token.

It might also be possible to set the store state (eg, roles array) so that the getter returns True.

Reply:
> It might also be possible to set the store state (eg, roles array) so that the getter returns True.

Unfortunately, that doesn't appear possible.  From what I've read, `rootState` is immutable -- which is probably a sensible decision by the Vuex developers.  I'll set up a mock.

Reply:
Is what you're asking about different than this?
`store.state.stateModel.businessId = 'BC1234567'`

Reply:
> Is what you're asking about different than this (which is allowed in the unit tests)? `store.state.stateModel.businessId = 'BC1234567'`
Yes, this is permitted:
 `store.state.stateModel.businessId = 'BC1234567'`
but attempting to change rootState is not -- it's immutable:
 `store.rootState.account.currentUser.roles = ['staff']`

Reply:
Ohhhhhh...... I've never tried assigning `store.rootState`.

Reply:
@severinbeauvais would it be possible to leave the replacing `isRoleStaff` to another ticket?  Moving the getter from our store to the Authorization module (or whatever we decide to call it) is trivial.  The time-consuming part is fixing our tests.  As discussed above we would have to change the following in each test.
`store.state.keycloakRoles = ['staff']`

At present, I don't have a single line replacement that can update state in another module.  I'm sure I can get this working, but I don't want to hold up the other work I'm doing.  Thoughts?

Reply:
Yes. Create another ticket. We have too many higher-priority things to work on.